lib/process-services/src/lib/task-list/services/task-filter.service.ts

```diff
@@ -57,50 +54,48 @@ export class TaskFilterService {
         const completeObservable = this.addFilter(completedTasksFilter);
 
         return new Observable((observer) => {
-            forkJoin([
-                    myTaskObservable,
-                    involvedObservable,
-                    queuedObservable,
-                    completeObservable
-                ]
-            ).subscribe(
-                (res) => {
-                    const filters: FilterRepresentationModel[] = [];
-                    res.forEach((filter) => {
-                        if (!this.isFilterAlreadyExisting(filters, filter.name)) {
-                            if (filter.name === involvedTasksFilter.name) {
-                                filters.push(new FilterRepresentationModel({
+            forkJoin([myTaskObservable, involvedObservable, queuedObservable, completeObservable]).subscribe((res) => {
+                const filters: FilterRepresentationModel[] = [];
+                res.forEach((filter) => {
+                    if (!this.isFilterAlreadyExisting(filters, filter.name)) {
+                        if (filter.name === involvedTasksFilter.name) {
```

Comment:
I think we can simplify this part by using switch case on `filter.name` setting `filter` variable and then after switch case block we could push new `FilterRepresentationModel` only once with correct params, since between those if else blocks then only changing variable is the `filter` itself. It would be quite a simpler and shorter this way

Reply:
Okay, fine for me